packages/core/package.json

```diff
@@ -5,6 +5,29 @@
   "license": "MIT",
   "main": "dist/use-gesture-core.cjs.js",
   "module": "dist/use-gesture-core.esm.js",
+  "exports": {
+    ".": {
+      "types": "./dist/use-gesture-core.cjs.d.ts",
+      "module": "./dist/use-gesture-core.esm.js",
+      "default": "./dist/use-gesture-core.cjs.js"
+    },
+    "./actions": {
+      "types": "./dist/declarations/src/actions.d.ts",
+      "module": "./dist/use-gesture-core.esm.js",
+      "default": "./dist/use-gesture-core.cjs.js"
+    },
+    "./types": {
+      "types": "./dist/declarations/src/types.d.ts",
+      "module": "./dist/use-gesture-core.esm.js",
+      "default": "./dist/use-gesture-core.cjs.js"
+    },
+    "./utils": {
+      "types": "./dist/declarations/src/utils.d.ts",
+      "module": "./dist/use-gesture-core.esm.js",
+      "default": "./dist/use-gesture-core.cjs.js"
+    },
```

Comment:
All of those module and default conditions point to the same file - this is wrong, they should point to the files representing those entrypoints. It would be the easiest to generate this through preconstruct - even if you don't end up enabling its `exports`-related options just yet (u can enable them temporarily, generate this stuff and roll back those options)

Reply:
Yes, I just was asking that myself just now. I was of the impression use-gesture still worked in my repo with these patches but I think I didn't look hard enough.

Reply:
It works in my repo because I don't use the sub path imports myself in my code :). So, this PR would probably work for me but of course it is wrong.